first_django_project/discography/models.py

```diff
@@ -11,10 +10,20 @@ def __str__(self):
 
 
 class Track(models.Model):
-    name = models.CharField(max_length=100, validators=[MinLengthValidator(3, message="Name's len must be more than 2")])
+    name = models.CharField(max_length=100,
+                            validators=[MinLengthValidator(3, message="Name's len must be more than 2")])
     date_published = models.DateField()
     artist = models.ForeignKey(Artist, on_delete=models.CASCADE, related_name="tracks")
 
-
     def __str__(self):
         return f"{self.name}, artist - {self.artist}"
+
+
+class Comment(models.Model):
+    comment_text = models.CharField(max_length=1000,

```

Comment:
simple text. redundant comment. comment.comment_text?

Reply:
like in official tutorial 
![image](https://user-images.githubusercontent.com/106016944/172075282-f01412c0-07a1-4143-89e8-f9f0737c2ac1.png)